backend/poetry.lock

```diff
@@ -25,7 +25,7 @@ version = "1.4.4"
 [[package]]
 category = "dev"
 description = "Disable App Nap on OS X 10.9"
-marker = "sys_platform == \"darwin\""
+marker = "python_version >= \"3.4\" and sys_platform == \"darwin\" or sys_platform == \"darwin\""
```

Comment:
weird change, but I guess it auto-generated

Reply:
Wow, look what github  added!
```suggestion
wow a new feature = "python_version >= \"3.4\" and sys_platform == \"darwin\" or sys_platform == \"darwin\""
```

Reply:
I saw it on gitlab years ago...
But it's cool every time since then